aspnetcore/signalr/hubcontext.md

```diff
@@ -53,7 +53,7 @@ app.Use(async (context, next) =>
 ```
 
 > [!NOTE]
-> When hub methods are called from outside of the `Hub` class, there's no caller associated with the invocation. Therefore, there's no access to the `ConnectionId`, `Caller`, and `Others` properties.
+> When hub methods are called from outside of the `Hub` class, there's no caller associated with the invocation. Therefore, there's no access to the `ConnectionId`, `Caller` and other properties.
```

Comment:
The doc is talking about how inside the `Hub` you have access to `Clients.Others`, and here it says you do **not** have access to the `Others` property when you're outside the `Hub`.

https://docs.microsoft.com/dotnet/api/microsoft.aspnetcore.signalr.ihubcallerclients

Reply:
Hi BrennanConroy,  got it, thanks for pointing that.